ruleset/testing/tests/php.ini

```diff
@@ -0,0 +1,17 @@
+;  Copyright (C) 2015-2021, Wazuh Inc.
+;
+;  Tests for products:
+;    PHP
```

Comment:
Could you please provide more information about the software version and such?

Reply:
I don't have the faintest idea of which PHP version is this for as it's missing in the original rules